adapters/Utils/Utils/AdapterUtils.py

```diff
@@ -526,15 +522,15 @@ def read_image_dataset(config: "StartAutoMlRequest") -> Tuple[pd.DataFrame, pd.D
 
         local_dir_path = os.path.dirname(local_file_path)
     """
-    data_dir = config.file_location
+    data_dir = config["dataset_path"]
```

Comment:
@jnnks Technically, both are correct, it is the dictionary of StartAutoMlRequest. 
It might be better to just change it to dict, as it is serialized to be passed to the subprocess